libs/core/algorithms/include/hpx/parallel/algorithms/sort_by_key.hpp

```diff
@@ -47,15 +47,16 @@ namespace hpx { namespace parallel { inline namespace v1 {
     /// The algorithm is not stable, the order of equal elements is not guaranteed
     /// to be preserved.
     /// The function uses the given comparison function object comp (defaults
-    /// to using operator<()).
+    /// to using operator<()). Executed according to the policy.
     ///
     /// \note   Complexity: O(Nlog(N)), where N = std::distance(first, last)
     ///                     comparisons.
     ///
-    /// A sequence is sorted with respect to a comparator \a comp and a
-    /// projection \a proj if for every iterator i pointing to the sequence and
+    /// A sequence is sorted with respect to a comparator \a comp
+    /// if for every iterator i pointing to the sequence and
     /// every non-negative integer n such that i + n is a valid iterator
     /// pointing to an element of the sequence, and
+    /// FIXME(bhumit,hkaiser): proj doesn't exist
     /// INVOKE(comp, INVOKE(proj, *(i + n)), INVOKE(proj, *i)) == false.
```

Comment:
@hkaiser what should this be now that `proj` doesn't exist?

Reply:
`INVOKE(comp, *(i + n), *i) == false`